engine/table/src/main/java/io/deephaven/engine/table/impl/select/MatchFilter.java

```diff
@@ -390,6 +399,18 @@ Object convertStringLiteral(String str) {
                         }
                         return Byte.parseByte(str);
                     }
+
+                    @Override
+                    Object convertParamValue(Object paramValue) {
+                        paramValue = super.convertParamValue(paramValue);
+                        if (paramValue instanceof Byte) {
```

Comment:
`paramValue.getClass()` is going to NPE...
```suggestion
                        if (paramValue instanceof Byte || paramValue == null) {
```